### PR TITLE
Update init.mdx

### PR DESCRIPTION
**Pull request by kmchau428**
Suspect it is a typo...

### CODE REVIEW

**Comment by alisdair** [comment]
This is not a typo, `-force-copy` implies `-migrate-state`: https://github.com/hashicorp/terraform/blob/036db86c1fc7f64bc03ff75150e50456b28c9f2b/internal/command/init.go#L83-L87

Can you suggest a clearer way to word that?

**Comment by kmchau428** [comment]
> This is not a typo, `-force-copy` implies `-migrate-state`:
> 
> https://github.com/hashicorp/terraform/blob/036db86c1fc7f64bc03ff75150e50456b28c9f2b/internal/command/init.go#L83-L87
> 
> Can you suggest a clearer way to word that?

If I understand correctly, does that mean cloning the existing state to the new backend?

**Comment by alisdair** [comment]
That's what `-migrate-state` does, yes.

**Review by kmchau428 — COMMENTED** [review]
Made the intention of `-migrate-state` with `-force-copy` more explicit.

File: website/docs/cli/commands/init.mdx [hunk 1]
```diff
@@ -85,7 +85,7 @@ The `-migrate-state` option will attempt to copy existing state to the new
 backend, and depending on what changed, may result in interactive prompts to
 confirm migration of workspace states.  The `-force-copy` option suppresses
 these prompts and answers "yes" to the migration questions. This implies
-`-migrate-state`.
+initiating the new backend with the existing state.
```
Comment (alisdair) [inline website/docs/cli/commands/init.mdx:88]: I think there's a disconnect here between the intent of the original documentation and your proposal. What I'd suggest instead here to clarify the text which caused the misunderstanding is replacing the "This implies […]" sentence with something like:

> Enabling `-force-copy` also automatically enables the `-migrate-state` option.

**Review by alisdair — CHANGES_REQUESTED** [review]

**Review by alisdair — APPROVED** [review]
Thanks!

**Pull request merged**